xla/myplugin/myplugin_pjrt.cc

```diff
@@ -0,0 +1,46 @@
+#include "xla/pjrt/c/pjrt_c_api_wrapper_impl.h"
+#include "myplugin_pjrt.h"
+#include "myplugin_cpp_pjrt.h"
+
+namespace myplugin_pjrt {
+
+std::unique_ptr<xla::PjRtClient> GetPluginPjRtClient() {
+  return std::make_unique<myplugin_pjrt::MypluginPjrtClient>();
+}
+
+// Create my client
+PJRT_Error* PJRT_MypluginClient_Create(PJRT_Client_Create_Args* args) {
+  std::unique_ptr<xla::PjRtClient> client = GetPluginPjRtClient();
+  args->client = pjrt::CreateWrapperClient(std::move(client));
+  printf("Creating PJRT Client from myplugin_pjrt.cc\n");
+  return nullptr;
+}
+
+PJRT_Error* PJRT_MypluginExecuteContext_Create(PJRT_ExecuteContext_Create_Args* args) {
+  return new PJRT_Error{absl::UnimplementedError(
+      "ExecuteContext not supported for MyPlugin execution.")};
+}
+
+PJRT_Error* PJRT_MypluginDeviceTopology_Create(
+    PJRT_TopologyDescription_Create_Args* args) {
+  return new PJRT_Error{
+      absl::UnimplementedError("Topology not supported for MyPlugin compilation.")};
+}
+
+} // namespace myplugin_pjrt
+
+const PJRT_Api* GetPjrtApi() {
+  printf("C++ Calling GetPjrtApi");
+  static PJRT_Layouts_Extension layouts_extension =
```

Comment:
Is there a way / a need to clean up a plugin? With these vars declared as static and scoped to this function alone seems like no?